src/main/scala/org/broadinstitute/hail/driver/VEP.scala

```diff
@@ -254,11 +260,12 @@ object VEP extends Command {
     if (conservationFile == null)
       fatal("property `hail.vep.conservation_file' required")
 
-    val cmd = Array(
+    val cmd = if(csq)
+      Array(
       perl,
       s"$location",
       "--format", "vcf",
-      "--json",
+      "--vcf",
```

Comment:
Just put the if inside so you don't duplicate the entire argument list: `Array(..., if (csq) "--vcf" else "--json", ...)`.